src/__test__/utils.test.js

```diff
@@ -0,0 +1,130 @@
+const child_process = require("child_process");
+const tmp = require("tmp");
+const { copySync } = require("fs-extra");
+const path = require("path");
+const fs = require("fs");
+const semver = require("semver");
+
+const { runChecks, getTags, getCommands, getWorkspaces } = require("../utils");
+
+function addTag(tag) {
+    child_process.execSync(`git tag ${tag}`);
+}
+
+function bumpVersion(pkgPath, release) {
+    const pkgJsonPath = path.join(pkgPath, "package.json");
+    const json = JSON.parse(fs.readFileSync(pkgJsonPath));
+    const newVersion = semver.inc(json.version, release);
+    json.version = newVersion;
+    fs.writeFileSync(pkgJsonPath, JSON.stringify(json, null, 4));
+}
```

Comment:
So this is the step that we don't do with `antwerp`. `antwerp` assumes you've done all the version changes that you need to and then just ensure that things are tagged with those versions before publish, after asserting that the dependencies are available for those packages.

Is that right?